src/06-loops.js

```diff
@@ -61,8 +83,17 @@ function rangeEveryOther(min, max) {}
  *  containsWhileLoop([ "left", "up", "right" ], "down");
  *  //> false
  */
-function containsWhileLoop(array, target) {}
-
+ function containsWhileLoop(array, target) {
+  let i = 0;
+    while (i < array.length) {
+     if(array.indexOf(target) != -1){
```

Comment:
Here... our while loop isn't really doing the work for us. Instead, by using `array.indexOf(target)` we've outsourced our while loops' work because `.indexOf` does this iteration for us. TLDR- If you took out the `while` loop and just wrote the conditional you would still arrive to the correct outcome because our `while` loop is not really doing anything here. Consider rewriting this to check each element in the array to see if it is our target element...